code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -39,7 +39,7 @@
 	return ..()
 
 /datum/reagent/consumable/nutriment
-	name = "Nutriment"
+	name = "Liquid meat"
```

Comment:
so when I grind up a potato, I'm supposed to get liquid MEAT?

Reply:
also, make sure to capitalize the names of these chems like titles, to be consistent with other chems